src/database/sqlite/utils.rs

```diff
@@ -262,6 +333,11 @@ pub fn maybe_apply_migration(db_path: &path::Path) -> Result<(), SqliteDbError>
                 migrate_v3_to_v4(&mut conn)?;
                 log::warn!("Migration from database version 3 to version 4 successful.");
             }
+            4 => {
+                log::warn!("Upgrading database from version 4 to version 5.");
+                migrate_v4_to_v5(&mut conn, bitcoin_txs)?;
```

Comment:
nit: we could log the number of Bitcoin transactions to be inserted in DB at the same time?

Reply:
I added another `log::warn` message here.